src/generators/operationGenerator.ts

```diff
@@ -1098,6 +1099,17 @@ function addImports(
     operationGroupFile
   );
 
+  const interfaceName = normalizeName(
+    operationGroupDetails.name,
+    NameType.OperationGroup,
+    true /** shouldGuard */
+  );
```

Comment:
this is a redefinition of the `className` variable above. I think it is better to rename `className` to `interfaceName` and remove this redefinition here.

Reply:
Removed the interfaceName and used the className directly